src/src/components/form/CustomChoiceRadio.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <div :class="radioContainerClasses">         
+    <label
+        v-if="undefinedCheck(radio.prefix)"
+        v-html="undefinedCheck(radio.prefix)" />
+    <span
+        v-for="option in radio.options"
+        :key="fieldID(option)"
+        class="radio-choice-container"
+    >
+        <input
+            type="radio"
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 12 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        type="radio"
```